### DISCUSSION TITLE
Add optional download button for assets

### DISCUSSION THREAD

**Discussion by ann0see** [community/community#6163] [Other Feature Feedback, Questions, & Ideas]
Currently some projects use their release page for application downloads. Especially for new, non technical users the current - and new UI is challenging: having to scroll, maybe even click on "assets" to expand the release assets and then finding a download for their OS is not intuitive.

SourceForge is much better in handling binary downloads since they have a prominent download button.

I would suggest to add a download button or a more prominent way to download assets on the new release page.
![SourceForge download button](https://user-images.githubusercontent.com/20726856/136673297-f6608cf2-64c3-49dc-a39d-711bf955697e.png)

Maybe even something like that would be enough:

![DownloadAssets](https://user-images.githubusercontent.com/20726856/136673069-7ff19e3d-efee-4fed-8af9-2ecad16cba38.png)

and (although for sure aligned)
![DL Assets](https://user-images.githubusercontent.com/20726856/136673235-768c7e4f-66c4-4e79-a8d4-ce563ce30ec4.png)

**Comment by MylesBorins** [comment]
Thanks for the suggestion. We have definitely thought about ways that we could improve downloading a release, especially if we could make it a single button that is smart about the platform you are on when requesting the download.

This is not in scope for our current work but definitely is something we could explore in the future

**Reply by ychin** [reply]
Yeah I think what I really want from a GitHub release revamp is just to allow me to pick which asset is the one that users should pick and show a big "Download" button for that. Vast majority of users aren't going to ever want to download zip or tar.gz source code, or the Sparkle delta update binary. The only thing they want is the dmg file for a macOS app, and it's kind of confusing and cluttered to see so many options.
